tests/migrations/test_commands.py

```diff
@@ -521,24 +521,6 @@ def test_showmigrations_plan_multiple_app_labels(self):
             out.getvalue()
         )
 
-    @override_settings(INSTALLED_APPS=['migrations.migrations_test_apps.unmigrated_app'])
-    def test_showmigrations_plan_app_label_error(self):
-        """
-        `showmigrations --plan app_label` raises an error when no app or
-        no migrations are present in provided app labels.
-        """
-        # App with no migrations.
-        with self.assertRaisesMessage(CommandError, 'No migrations present for: unmigrated_app'):
```

Comment:
I don't think this first test situation should be deleted and the new behavior should match the new test cases (stderr instead of CommandError) -- `test_showmigrations_unmigrated_app` should be updated similarly

Reply:
oh, I didn't see the 'plan'. I see. I will change it soon.